README.md

```diff
@@ -125,6 +125,11 @@ Options:
                                pattern (default is false)
       --sf, --separateFiles  generate a separate file for each class (default
                                is false)
+      --dnfin, --noNotForceIsNullable
```

Comment:
This should probably be `--doNotForceIsNullable`.

Reply:
Should be fixed now.
Thanks for the review!